app/src/main/java/com/android/sample/ui/composables/WorkoutSummaryScreen.kt

```diff
@@ -0,0 +1,117 @@
+package com.android.sample.ui.composables
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.ui.theme.Green
+import com.android.sample.ui.theme.LightGrey
+import com.android.sample.ui.theme.Red
+import com.android.sample.ui.workout.ExerciseState
+
+/**
+ * Composable function that displays a summary screen for a workout.
+ *
+ * This screen shows a summary of the completed workout, including:
+ * - A card indicating whether a warm-up was done.
+ * - A list of exercise cards, each displaying the exercise type and a checkmark if completed.
+ * - A finish button to navigate away from the summary screen.
+ *
+ * @param hasWarmUp A boolean value indicating whether the workout included a warm-up.
+ * @param exerciseList A list of [ExerciseState] objects representing the exercises in the workout.
+ * @param onfinishButtonClicked A lambda function to be invoked when the finish button is clicked.
+ */
+@Composable
+fun WorkoutSummaryScreen(
+    hasWarmUp: Boolean,
+    exerciseList: List<ExerciseState>,
+    onfinishButtonClicked: () -> Unit
+) {
+  Column(
```

Comment:
There is a problem if you have a lot of Exercises, you won't be able to see them all and you can't scroll. You need to add "scrollability" to your Column with like `val scrollState = rememberScrollState()` and `modifier = Modifier
            .verticalScroll(scrollState)`
            
            Or you can use a LazyColumn

Reply:
This is managed by the caller. (WorkoutScreen) wich places them in a scrollable collumn. The case where there is a lot of exercise has been tested locally and works. Unfortunately placing a scrollable collumn (as you said) inside another scrollable collumns fails in Jetpack.

Reply:
Oh yeah you're right, didn't realize it was placed inside a scrollable column, and didn't know it would work